docs/root/intro/version_history.rst

```diff
@@ -60,6 +60,11 @@ Version history
 * network filters: added a :ref:`direct response filter <config_network_filters_direct_response>`.
 * network filters: network filter extensions use the "envoy.filters.network" name space. A mapping
   of extension names is available in the :ref:`deprecated <deprecated>` documentation.
+* network filters: added a :ref:`direct response filter <config_network_filters_direct_response>`.
```

Comment:
What's going on here with these extra lines?

Reply:
Conflict during merge. Thanks for spotting it.